tes3mp-deploy.sh

```diff
@@ -187,6 +188,12 @@ else
       fi
     ;;
 
+    # Build master server
+    -m | --build-master )
+      BUILD_MASTER=true
+      touch .buildmaster
```

Comment:
why do you need to make a file to determine if we build? Why not just use the same variable below?

Reply:
It's what the original version of deploy did, and it works so I just left it in.
https://github.com/GrimKriegor/TES3MP-deploy/commit/572ec056c72d9884d939b7cdbb9c5559a8305a1f